### ISSUE TITLE
Windows / Wsl

### ISSUE THREAD

**Issue by SonGokussj4** [trin94/PySide6-project-template#11]
I would really loved to try this template but don't know how. I'm using Windows10 with WSL2.

When I create and activate (Windows)
```
python3.10.exe -m venv venv
venv/Scripts/activate
python -m install -r requirements.txt
python main.py
```

it won't work, because 

```
No module named 'myapp.generated_resources'
```

When I try that On (Windows WSL)
```
python3.10 -m venv .venv
source .venv/bin/activate
python -m install -r requirements.txt`
```

it fails on installing PySide6 version you specified. So when I install PySide6 without specifying the version, then 

```
python main.py
```

It crashes because my GLIBC lib is 2.27 and it needs 2.28 and even `sudo apt upgrade` doesn't help.

### In Summary
- Can't run it on Windows because Makefile is written for Linux
- Can't run it on WSL because GLIBC is old there? Not sure

**Comment by trin94** [comment]
Hi,

Have you used **git-bash** in order to run the make build commands? As you mentioned, these commands might not work properly on Powershell, for instance.

Could you please try running the following commands in **git-bash** and let me know if they work for you? 😃 

```bash
git clone https://github.com/trin94/PySide6-project-template.git
cd PySide6-project-template
python -m venv venv
source venv/Scripts/activate
python -m pip install -r requirements.txt
make develop-build
python main.py
```

Feel free to reach out if you encounter any issues or have any questions.

Best regards,
Elias

**Comment by SonGokussj4** [comment]
Tried with **git-bash** but there was a problem
```
$ python3.10 -m venv venv
$ source venv/bin/activate  # <-- Documentation says `source venv/Scripts/activate` but git-bash creates "linux" style venv/bin/python
$ python -m pip install -r requirements.txt

$ make develop-build
/mnt/c/GIT/PySide6-project-template/venv/bin/python3
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-lupdate
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-lrelease
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-rcc

/usr/bin/env: ‘python\r’: No such file or directory
make: *** [Makefile:287: xtask-generate-qrc-i18n] Error 127
```

I know this error so I've found all files with `/usr/bin/env` and resaved **CRLF** as **LF**

```
/mnt/c/GIT/PySide6-project-template/main.py
/mnt/c/GIT/PySide6-project-template/build-aux/generate-lupdate-project-file.py
/mnt/c/GIT/PySide6-project-template/build-aux/generate-qt-creator-project-file.py
```

Then the build successfully succeeded.
```
$ make develop-build
/mnt/c/GIT/PySide6-project-template/venv/bin/python3
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-lupdate
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-lrelease
/mnt/c/GIT/PySide6-project-template/venv/bin/pyside6-rcc

Updating 'i18n/de_DE.qm'...
    Generated 4 translation(s) (4 finished and 0 unfinished)
    Ignored 10 untranslated source text(s)
Updating 'i18n/he_IL.qm'...
    Generated 4 translation(s) (4 finished and 0 unfinished)
    Ignored 10 untranslated source text(s)
```

But when running the app:
```
$ python main.py
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vkkhrdisplay, vnc, wayland-egl, wayland, xcb.

Aborted
```

**Comment by trin94** [comment]
Hi

This 
```
Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vkkhrdisplay, vnc, wayland-egl, wayland, xcb.
```

makes me think that Python/Qt assumes that you're on a Linux system. This line might be the reason for that

```shell
$ source venv/bin/activate  # <-- Documentation says `source venv/Scripts/activate` but git-bash creates "linux" style venv/bin/python
```

Have you tried using 

```shell
python3.10.exe -m venv venv
venv/Scripts/activate
```

and then continuing from there?

How have you installed Python on your system? Via the Microsoft Store? I'd recommend installing it manually from the official page: https://www.python.org/downloads/

Best regards,
Elias

**Comment by trin94** [comment]
I just ported the project to `just`. If the issue still persists, please open another issue :smile:
